client.lua

```diff
@@ -0,0 +1,50 @@
+local QBCore = exports['qb-core']:GetCoreObject()
+local hasPhone = false
+
+local function DoPhoneCheck(PlayerItems)
+    hasPhone = false
+    
+    if PlayerItems then
+        for _,item in pairs(PlayerItems) do
+            if Config.PhoneList[item.name] then
+                hasPhone = true
+                break;
+            end
+        end
+    end
+
+    exports['npwd']:setPhoneDisabled(not hasPhone)
+end
```

Comment:
Small indentation thing

```suggestion
local function DoPhoneCheck(PlayerItems)
    hasPhone = false

    if not PlayerItems then return end

    for _,item in pairs(PlayerItems) do
        if Config.PhoneList[item.name] then
            hasPhone = true
            break;
        end
    end

    exports['npwd']:setPhoneDisabled(not hasPhone)
end
```

Reply:
Done